server/config/policies.js

```diff
@@ -36,7 +36,7 @@ module.exports.policies = {
   "forms/post-contact-us-form": true,
 
   // Cache
-  "cache/*": ["isLoggedIn", "isAdmin"],
+  "cache/*": ["isLoggedIn", "isTech"],
```

Comment:
since isTech.js is actually checking if the accestype is "t3ch", "admin" or "stewardship", it is best to rename the function to reflect that. for example "isTechOrAbove". otherwise we might mistake it for checking if accestype is "tech" only.

Reply:
nice